Makefile

```diff
@@ -329,6 +354,25 @@ endif
 	@curl -Lo $(KUBECTL) --fail --silent https://dl.k8s.io/release/v$(KUBECTL_VERSION)/bin/${uname_s}/${COMMON_ARCH}/kubectl
 	@chmod +x $(KUBECTL)
 
+.PHONY: helm
```

Comment:
Where is helm used? Do not add unneeded dependencies to external binaries!

Reply:
Helm was being used in the grafana dashboards repo so we added it because we thought we needed it, but ended up not needing it. I will comment out or delete as per your request